pom.xml

```diff
@@ -2160,7 +2160,7 @@
             <configuration>
               <!-- Ignore runtime-only dependencies in analysis -->
               <ignoreNonCompile>true</ignoreNonCompile>
-              <failOnWarning>true</failOnWarning>
+              <failOnWarning>false</failOnWarning>
```

Comment:
Why was this needed. Can we update the jar without disabling 'failOnWarning' for all warnings ?

Reply:
For some reason fails locally with

```
[INFO] --- maven-dependency-plugin:3.0.2:analyze-only (default) @ beam-sdks-java-extensions-google-cloud-platform-core ---
[WARNING] Unused declared dependencies found:
[WARNING]    com.fasterxml.jackson.core:jackson-databind:jar:2.8.9:compile
```

reverted